Airports/EGPH/SMR/Regions.txt

```diff
@@ -166,6 +161,46 @@ N055.57.12.797 W003.21.16.420
 N055.57.12.852 W003.21.16.055
 N055.57.12.979 W003.21.15.516
 N055.57.13.802 W003.21.12.971
+; - Terminal_Building
+Terminal_Building 17003 N055.56.51.439 W003.21.56.447
```

Comment:
Needs to start with "REGIONNAME Edinburgh "

The colour is defined here: master/Colours.txt  and can be "terminal" to match the other parts of the SMR